x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -413,7 +413,7 @@ const AlertsTable: React.FunctionComponent<AlertsTableProps> = memo((props: Aler
       rowSelection: bulkActionsState.rowSelection,
       alerts,
       isLoading,
-      columnIds: visibleColumns,
+      columnIds: columns.map((column) => column.id),
```

Comment:
could you please add a test to ensure it doesn't happen again?

Reply:
Yep, I asked for help from your team on this above:

> @elastic/response-ops Got any ideas on how to best add unit tests for this?

Testing this isn't straight forward, at least not to me since I'm not familiar with the whole testing architecture around the alerts table. I would greatly appreciate your input here :)

Reply:
Sorry I missed the message.

Maybe a test that updates the browserField prop using rtl's rerender method in  `x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.test.tsx` ?

Reply:
@jcger Wdyt about https://github.com/elastic/kibana/pull/197043/commits/25b3a21cb8908b7497ee9ff44b2c0c8784b84755 ?